DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/_RegistrationPromptAnswerTable.cshtml

```diff
@@ -1,7 +1,10 @@
 ﻿@using DigitalLearningSolutions.Web.Controllers.TrackingSystem.CentreConfiguration
-@model List<string>
+@model (List<string>, bool)

```

Comment:
Could we create a new model for this to make things a bit neater now that we have different fields too?

Reply:
Added a new property to the base model and just passing the whole thing in instead. Seems cleaner than creating a whole new model that I would have to populate somewhere

Reply:
That's neat! I wonder if we could call the boolean "IncludeAnswersTableCaption" or something similar to make it clear what it does in the UI?